.buildkite/premerge.steps.yaml

```diff
@@ -30,17 +20,10 @@ steps:
     build:
       branch: *ci_version
       message: "gdk-4.26 ${BUILDKITE_MESSAGE}"
-      env: 
-        BUILD_TYPE: "GDK"
-        GDK_BRANCH: "${BUILDKITE_BRANCH}"
-        ENGINE_BRANCH: "4.26-SpatialOSUnrealGDK"
+      env:
+        <<: *common_env_vars
+        ENGINE_BRANCH: "${ENGINE_BRANCH_426:-4.26-SpatialOSUnrealGDK}"
```

Comment:
Ohhhh I see now, I thought this would need corresponding code in the new CI but I was broken yesterday. Yeah since we convert both ENGINE_BRANCH_426 and ENGINE_BRANCH_425 to ENGINE_BRANCH before it goes to new CI this is fine. Nice stuff dude